subprojects/language-native/src/main/java/org/gradle/language/nativeplatform/internal/AbstractNativeCompileSpec.java

```diff
@@ -62,6 +63,21 @@ public void include(Iterable<File> includeRoots) {
         addAll(this.includeRoots, includeRoots);
     }
 
+    @Override
+    public List<File> getSystemIncludeRoots() {
+        return systemIncludeRoots;
+    }
+
+    @Override
+    public void systemInclude(Iterable<File> systemIncludeRoots) {
+        addAll(this.systemIncludeRoots, systemIncludeRoots);
+    }
+
+    @Override
+    public void systemInclude(File... systemIncludeRoots) {
```

Comment:
Aren't we always going to use the `Iterable` version above?

Reply:
Yeah, I was just making it consistent with `include()`.  I've removed it.